course/sandbox.py

```diff
@@ -344,4 +345,35 @@ def make_form(data=None):
 # }}}
 
 
+# {{{ remove common indentation of content in SandBox Form
+
+def remove_sb_content_common_indentation(code):
+    if "\n" not in code:
+        return code
+
+    lines = code.split("\n")
+
+    if lines:
+        base_indent = 0
+
+        # form field with textarea widget returns 
```

Comment:
Maybe I am wrong in that the first line is stripped by codemirror.CodeMirrorTextarea?